### PR TITLE
Catch exception of historical folo record and exit

### PR DESCRIPTION
**Pull request by yma96new**

### CODE REVIEW

**Comment by yma96new** [comment]
Ref: https://issues.redhat.com/browse/MMENG-4222
@sswguo, @ligangty I'm Ying, this is a temp account I use to commit. I try to recovery my original github account token which was lost.
Could you help to add review, and help to merge this if no problem around? I don't have access to do these, thanks.

File: src/main/java/org/commonjava/util/sidecar/services/ReportService.java [hunk 1]
```diff
@@ -119,12 +121,31 @@ public void storeTrackedDownload( JsonObject message )
...
-        Response response = trackingService.recordArtificat( trackingId, trackingPath, storeKey.getPackageType(),
-                                                             storeKey.getType().name(), storeKey.getName(), originalUrl,
-                                                             entryDTO.getSize(), entryDTO.getMd5(), entryDTO.getSha1(),
-                                                             entryDTO.getSha256() );
-        logger.debug( "Finished consuming folo record seal event for path:{}, trackingId:{}, rep code: {}, body: {}",
-                      trackingPath, trackingId, response.getStatus(), response.getStatusInfo().getReasonPhrase() );
+        boolean exception = false;
+        try
+        {
+            Response response = trackingService.recordArtificat( trackingId, trackingPath, storeKey.getPackageType(),
```
Comment (sswguo) [inline src/main/java/org/commonjava/util/sidecar/services/ReportService.java:127]: Typo on `recordArtificat`.

Comment (yma96) [reply src/main/java/org/commonjava/util/sidecar/services/ReportService.java:127]: @sswguo sorry, miss this comment, fix on PR: https://github.com/Commonjava/indy-sidecar/pull/83, which I use my original recovered account, please review, thanks.

**Comment by yma96** [comment]
Merged: https://github.com/Commonjava/indy-sidecar/pull/83

**Pull request merged**